SalesforceCommerceCloud/pwa-kit
# Fix Redirect After Account Creation

# Description

This PR fixes the redirect that happens after creating a new account. This was previously always using the current locale in the path, but should not be using it if the configuration for locales is set to not display in the url.

To test his change do the following 2 actions: 

1. Create a new account and ensure you are redirected correctly (when running the PWA project in the mono repo)
2. Create a new account and ensure you are redirected correctly (after generating a project using the demo settings)
